src/test/Fake.Core.UnitTests/Fake.Core.Process.fs

```diff
@@ -11,14 +11,15 @@ let fsCheckConfig = { FsCheckConfig.defaultConfig with maxTest = 1000  }
 [<Tests>]
 let tests = 
   testList "Fake.Core.Process.Tests" [
-    testPropertyWithConfig fsCheckConfig "toWindowsCommandLine is the inverse of fromWindowsCommandLine" <|
+    //Process.setEnableProcessTracing true
```

Comment:
If this line is uncommented it completely breaks the "fake-context" for some reason and about the half (around 30) tests start to fail... Would be interesting to figure out why.
Note that not only tests in this file start failing but everywhere (especially the `FakeVar` and `Context` ones)